api/src/pcapi/routes/backoffice/collective_offers/collective_offers_blueprint.py

```diff
@@ -102,6 +102,16 @@
         "column": educational_models.CollectiveOffer.venueId,
     },
     "VALIDATION": {"field": "validation", "column": educational_models.CollectiveOffer.validation},
+    "PRICE": {
+        "field": "price",
+        "column": aliased_stock.price,
+        "inner_join": "stock",
+        "arbitrary_filter": sa.and_(
+            aliased_stock.isEventExpired == False,
+            aliased_stock.hasBookingLimitDatetimePassed == False,
+            educational_models.CollectiveOffer.isActive == True,
```

Comment:
On utilisera plutôt : 
```suggestion
            aliased_stock.isEventExpired.is_(False),
            aliased_stock.hasBookingLimitDatetimePassed.is_(False),
            educational_models.CollectiveOffer.isActive.is_(True),
```
ou encore : 
```suggestion
            sa.not_(aliased_stock.isEventExpired),
            sa.not_(aliased_stock.hasBookingLimitDatetimePassed),
            educational_models.CollectiveOffer.isActive,
```

Reply:
collective_offers/blueprint ligne 110 a 112:
```
aliased_stock.isEventExpired.is_(False),
aliased_stock.hasBookingLimitDatetimePassed.is_(False),
educational_models.CollectiveOffer.isActive.is_(True),
```
FAIT